tests/pin_validator.test.ts

```diff
@@ -0,0 +1,17 @@
+import isOibValid from "../src/components/utils/pin_validator";
+
+describe("isOibValid function", () => {
+  it("incorrect OIB length", () => {
+    expect(isOibValid(1)).toBe(false);
+    expect(isOibValid(1234567891012)).toBe(false);
+  });
+  it("valid OIB", () => {
```

Comment:
Small nit comment, but there is a missing enter before `it("valid OIB", () => {`.